### PR TITLE
WIP: Add description of HCM 

### PR DESCRIPTION
**Pull request by z357412526**

### CODE REVIEW

**Comment by shamindras** [comment]
@z357412526 - what is the status on this one?

The [checks are currently failing](https://travis-ci.org/berkeley-stat222/mousestyles/jobs/121099538). They seem to be PEP8 errors i.e. Python formatting that is not consistent with the PEP 8 standard.

The error messages guide you into what these issues are. Are you able to make changes, commit, pull-request, review and send a MRG request once all tests are passing?

**Comment by peterasujan** [comment]
@shamindras @z357412526 I think these are old style errors from before @jarrodmillman fixed them in #49. A rebase should get things up-to-date and passing.

**Comment by jarrodmillman** [comment]
@z357412526 , what is the status of this?

**Comment by z357412526** [comment]
Sorry, I did not understand the problem. Do we need to rebase it?

File: doc/problem/behavior.md [hunk 1]
```diff
@@ -9,7 +9,9 @@ In order to differentiate mice, we can create behavior profile to describe
...
+- **Home Cage Monitoring System(HCM)** HCM cages were spatially discretized into a 12 x 24 array of cells and occupancy times for each MD were computed as the proportion of time spent within each of the 288 cells. To determine whether animals establish a Home base, HCM cages were spatially discretized into a 2 x 4 array of cells, and occupancy times for each mouse day were calculated as above. In the experiment, 56/158 animals displayed largest occupancy times in the cell containing the niche area, which was considered to be their Home base location.
```
Comment (jarrodmillman) [inline doc/problem/behavior.md]: My understanding of the home cage monitoring system was that it is the system that all our data is coming from.  I believe we have some data which is not occupancy times in a 12 x 24 grid.

Also the home base and niche are different.  It just turns out in practice that home base is often established in the niche.  I believe some mice made home bases outside the niche.

Please investigate further and rewrite this.

**Comment by jarrodmillman** [comment]
As I mentioned in class, I tend to rebase frequently, while working on a pull request.  You can figure out what a good practice is for you.

I will require that you rebase before I merge your pull request though.

I made a few comments on the text, but your work still needs to be reviewed.  Please get someone in the class to review your work more thoroughly.

**Comment by shamindras** [comment]
@z357412526 - this looks like it just has not been `rebased` correctly. It has some basic python `PEP8` formatting errors failing due to the older data loading code.

Could you please follow the `rebase` instructions in [CONTRIBUTING](https://github.com/berkeley-stat222/mousestyles/blob/master/CONTRIBUTING.rst) to rebase and push to origin so that all the tests pass.

If you need help, please feel free to chat on [our gitter channel](https://gitter.im/berkeley-stat222/mousestyles) and someone will help out 😄 !

**Comment by jarrodmillman** [comment]
This pull request is from your master branch.  You need to use a branch other than master.  I can help you fix this in class tomorrow.

**Pull request closed without merging**